pkg/controllers/provisioning/scheduling/instance_selection_test.go

```diff
@@ -43,8 +43,8 @@ var _ = Describe("Instance Type Selection", func() {
 	var minPrice float64
 	var instanceTypeMap map[string]*cloudprovider.InstanceType
 	nodePrice := func(n *v1.Node) float64 {
-		of, _ := instanceTypeMap[n.Labels[v1.LabelInstanceTypeStable]].Offerings.Get(n.Labels[v1beta1.CapacityTypeLabelKey], n.Labels[v1.LabelTopologyZone])
-		return of.Price
+		return instanceTypeMap[n.Labels[v1.LabelInstanceTypeStable]].Offerings.Compatible(
```

Comment:
Any reason to not use `Get` here, and use `Cheapest`? Only one offering should match since well known labels have already been resolved on the node right?